vectorizers/tests/test_common.py

```diff
@@ -391,26 +403,3 @@ def test_wass1d_transfomer():
                 kantorovich1d(histogram_data[i], histogram_data[j]),
                 np.sum(np.abs(result[i] - result[j])),
             )
-
-
-def test_node_removal():
-    graph = scipy.sparse.random(10, 10, 0.1, format="csr")
-    node_to_remove = np.argmax(np.array(graph.sum(axis=0)).T[0])
-    graph_less_node = remove_node(graph, node_to_remove, inplace=False)
-    assert (graph != graph_less_node).sum() > 0
-    with pytest.raises(ValueError):
-        graph_less_node = remove_node(graph, node_to_remove, inplace=True)
-    inplace_graph = graph.tolil()
```

Comment:
It looks like your merge in the test_common accidentally removed Lelands test_node_removal

Reply:
Thanks for catching that Colin.  I've added it back in.